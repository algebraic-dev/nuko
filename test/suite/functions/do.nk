let main =
    let b = 3
    print b
    print "b"