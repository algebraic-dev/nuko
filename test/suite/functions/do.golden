let main = do 
    print "a"
    print "b"